src/drag-and-drop/components/container.tsx

```diff
@@ -63,7 +86,20 @@ export const Container: React.FC<IProps> = ({ authoredState, interactiveState, s
         };
       }
     });
-  }, [authoredState.draggableItems]);
+    authoredState.dropZones?.forEach(target => {
+      if (target.imageUrl) {
+        const img = document.createElement("img");
+        img.src = target.imageUrl;
+        img.onload = () => {
+          setTargetDimensions(prevHash => ({...prevHash, [target.id]: {width: target.targetWidth || img.width, height: target.targetHeight || img.height }}));
+        };
+      } else {
+        setTargetDimensions(prevHash => (
+          {...prevHash, [target.id]: {width: target.targetWidth || 100, height: target.targetHeight  || 100 }}
```

Comment:
`{...prevHash, [target.id]: {width: target.targetWidth || 100, height: target.targetHeight  || 100}}`

Reply:
make sure spacing around brackets is the same

Reply:
And a note here, you can either have the space at the beginning/end or not have the space.  Doesn't matter as long as it is consistent.  I would follow whatever is most common in the project.